core/lib/dal/src/eth_sender_dal.rs

```diff
@@ -291,6 +291,45 @@ impl EthSenderDal<'_, '_> {
         Ok(stats)
     }
 
+    /// This query returns the number and sent_at_block of the latest commited batch.
+    /// Query should be primary used for L1, not for gateway settlement layer.
+    ///
+    /// Although, here we are not checking if the batch is from gateway settlement layer,
```

Comment:
why? The chain might've just migrated to L1 and so the last batch has been committed to GW